model/actors/miner/task.go

```diff
@@ -41,6 +47,15 @@ type MinerTaskResult struct {
 func (mtr *MinerTaskResult) Persist(ctx context.Context, db *pg.DB) error {
 	ctx, span := global.Tracer("").Start(ctx, "MinerTaskResult.Persist")
 	defer span.End()
+
+	ctx, _ = tag.New(ctx, tag.Upsert(metrics.TaskNS, tasks.MinerPoolName))
+	stats.Record(ctx, metrics.TaskQueueLen.M(-1))
```

Comment:
:question: If this is tagged before creating the trace above, will the trace have the tags on it?

Reply:
Yes it will, I can move it above if you would like?